platform-sdk/swirlds-jasperdb/src/test/java/com/swirlds/merkledb/files/DataFileReaderCloseTest.java

```diff
@@ -48,11 +48,12 @@ class DataFileReaderCloseTest {
 
     private static final DataItemSerializer<long[]> serializer = new TwoLongSerializer();
 
+    private static MerkleDbConfig dbConfig = ConfigurationHolder.getConfigData(MerkleDbConfig.class);
+
     @BeforeAll
     static void setup() throws IOException {
         final Path dir = TemporaryFileBuilder.buildTemporaryFile("readerIsOpenTest");
-        collection = new DataFileCollection<>(
-                ConfigurationHolder.getConfigData(MerkleDbConfig.class), dir, "store", serializer, null);
+        collection = new DataFileCollection<>(dbConfig, dir, "store", serializer, null);
```

Comment:
Is it better to initialize `static dbConfig` here (even if for each test)?

Reply:
Done